source/ranges.tex

```diff
@@ -4552,15 +4552,15 @@
     { return ranges::begin(@\exposid{base_}@); }
 
     constexpr auto begin() const
-      requires range<const V> &&
+      requires @\libconcept{range}@<const V> &&
                indirect_unary_predicate<const Pred, iterator_t<const V>>
     { return ranges::begin(@\exposid{base_}@); }
 
     constexpr auto end() requires (!@\exposconcept{simple-view}@<V>)
     { return @\exposid{sentinel}@<false>(ranges::end(@\exposid{base_}@), addressof(*@\exposid{pred_}@)); }
 
     constexpr auto end() const
-      requires range<const V> &&
+      requires @\libconcept{range}@<const V> &&
                indirect_unary_predicate<const Pred, iterator_t<const V>>
```

Comment:
```suggestion
               @\libconcept{indirect_unary_predicate}@<const Pred, iterator_t<const V>>
```
There's 2 `indirect_unary_predicate` in master needing this change.

Reply:
fixed